apps/dotcom/client/src/tla/app/TldrawApp.ts

```diff
@@ -45,6 +46,9 @@ export class TldrawApp {
 
 	private constructor(store: Store<TldrawAppRecord>) {
 		this.store = store
+		if (isDevelopmentEnv) {
+			;(window as any).tla_dev_clear = () => this.store.clear()
```

Comment:
We expose this for e2e tests. Just resting the db was not enough since things still stayed in memory.

Think this should do for now, but I did notice some issues with running the e2e with the UI mode. In some cases the worker got stuck. If we find it problematic I can also look for an alternative solution.